<ui version="4.0" >
 <class>WizzardExport</class>
 <widget class="QDialog" name="WizzardExport" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>583</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Wizzard Export</string>
  </property>
  <property name="modal" >
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="layoutWidget" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>20</y>
     <width>561</width>
     <height>381</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" >
    <item>
     <layout class="QVBoxLayout" >
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QLabel" name="label" >
          <property name="text" >
           <string>Type of Ouput:</string>
          </property>
          <property name="buddy" >
           <cstring>cbtypeofoutput</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cbtypeofoutput" />
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>80</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <property name="spacing" >
         <number>4</number>
        </property>
        <item>
         <layout class="QVBoxLayout" >
          <property name="spacing" >
           <number>0</number>
          </property>
          <item>
           <widget class="QGroupBox" name="groupBox" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize" >
             <size>
              <width>0</width>
              <height>150</height>
             </size>
            </property>
            <property name="maximumSize" >
             <size>
              <width>220</width>
              <height>150</height>
             </size>
            </property>
            <property name="acceptDrops" >
             <bool>false</bool>
            </property>
            <property name="title" >
             <string>Content</string>
            </property>
            <widget class="QWidget" name="layoutWidget" >
             <property name="geometry" >
              <rect>
               <x>20</x>
               <y>40</y>
               <width>171</width>
               <height>89</height>
              </rect>
             </property>
             <layout class="QVBoxLayout" >
              <item>
               <layout class="QHBoxLayout" >
                <item>
                 <widget class="QRadioButton" name="rbAll" >
                  <property name="maximumSize" >
                   <size>
                    <width>167</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="layoutDirection" >
                   <enum>Qt::RightToLeft</enum>
                  </property>
                  <property name="text" >
                   <string>All</string>
                  </property>
                  <property name="checked" >
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer>
                  <property name="orientation" >
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeType" >
                   <enum>QSizePolicy::Fixed</enum>
                  </property>
                  <property name="sizeHint" >
                   <size>
                    <width>75</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" >
                <item>
                 <widget class="QRadioButton" name="rbSelected" >
                  <property name="maximumSize" >
                   <size>
                    <width>167</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="layoutDirection" >
                   <enum>Qt::RightToLeft</enum>
                  </property>
                  <property name="text" >
                   <string>Selection</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer>
                  <property name="orientation" >
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeType" >
                   <enum>QSizePolicy::Fixed</enum>
                  </property>
                  <property name="sizeHint" >
                   <size>
                    <width>75</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" >
                <property name="spacing" >
                 <number>0</number>
                </property>
                <property name="leftMargin" >
                 <number>9</number>
                </property>
                <item>
                 <widget class="QLabel" name="label_2" >
                  <property name="sizePolicy" >
                   <sizepolicy vsizetype="Preferred" hsizetype="Fixed" >
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="text" >
                   <string>id-to-id:</string>
                  </property>
                  <property name="buddy" >
                   <cstring>idtoidfield</cstring>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="idtoidfield" >
                  <property name="sizePolicy" >
                   <sizepolicy vsizetype="Preferred" hsizetype="Fixed" >
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="maximumSize" >
                   <size>
                    <width>70</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="toolTip" >
                   <string>id-id</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" >
            <property name="spacing" >
             <number>0</number>
            </property>
            <item>
             <spacer>
              <property name="orientation" >
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeType" >
               <enum>QSizePolicy::MinimumExpanding</enum>
              </property>
              <property name="sizeHint" >
               <size>
                <width>20</width>
                <height>67</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <spacer>
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType" >
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" >
               <size>
                <width>107</width>
                <height>72</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_2" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="title" >
           <string>Fields</string>
          </property>
          <layout class="QHBoxLayout" >
           <item>
            <layout class="QVBoxLayout" >
             <item>
              <layout class="QVBoxLayout" >
               <item>
                <widget class="QRadioButton" name="rbscreen" >
                 <property name="text" >
                  <string>As the screen</string>
                 </property>
                 <property name="checked" >
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="chosen" >
                 <property name="text" >
                  <string>Chosen Colunms</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" >
               <property name="spacing" >
                <number>-1</number>
               </property>
               <item>
                <widget class="QListWidget" name="listofallfield" >
                 <property name="enabled" >
                  <bool>false</bool>
                 </property>
                 <property name="sizePolicy" >
                  <sizepolicy vsizetype="Expanding" hsizetype="Minimum" >
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="maximumSize" >
                  <size>
                   <width>100</width>
                   <height>16777215</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item>
                <layout class="QVBoxLayout" >
                 <property name="spacing" >
                  <number>-1</number>
                 </property>
                 <item>
                  <widget class="QPushButton" name="addButton" >
                   <property name="sizePolicy" >
                    <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
                     <horstretch>1</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="text" >
                    <string>Add</string>
                   </property>
                   <property name="icon" >
                    <iconset resource="../playlistgenerator.qrc" >:/resources/pixmaps/next.png</iconset>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="removeButton" >
                   <property name="text" >
                    <string>Remove</string>
                   </property>
                   <property name="icon" >
                    <iconset resource="../playlistgenerator.qrc" >:/resources/pixmaps/previous.png</iconset>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <spacer>
                   <property name="orientation" >
                    <enum>Qt::Vertical</enum>
                   </property>
                   <property name="sizeHint" >
                    <size>
                     <width>20</width>
                     <height>111</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QListWidget" name="listofselectedfield" >
                 <property name="enabled" >
                  <bool>false</bool>
                 </property>
                 <property name="sizePolicy" >
                  <sizepolicy vsizetype="Expanding" hsizetype="Minimum" >
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize" >
                  <size>
                   <width>0</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize" >
                  <size>
                   <width>100</width>
                   <height>1960</height>
                  </size>
                 </property>
                 <property name="sortingEnabled" >
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" >
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>171</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QDialogButtonBox" name="buttonBox" >
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="standardButtons" >
         <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
        </property>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>171</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>cbtypeofoutput</tabstop>
  <tabstop>rbAll</tabstop>
  <tabstop>rbSelected</tabstop>
  <tabstop>idtoidfield</tabstop>
  <tabstop>rbscreen</tabstop>
  <tabstop>chosen</tabstop>
  <tabstop>listofallfield</tabstop>
  <tabstop>addButton</tabstop>
  <tabstop>removeButton</tabstop>
  <tabstop>listofselectedfield</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources>
  <include location="../playlistgenerator.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>WizzardExport</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>258</x>
     <y>384</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>WizzardExport</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>326</x>
     <y>390</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chosen</sender>
   <signal>clicked(bool)</signal>
   <receiver>listofallfield</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>310</x>
     <y>105</y>
    </hint>
    <hint type="destinationlabel" >
     <x>332</x>
     <y>142</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chosen</sender>
   <signal>toggled(bool)</signal>
   <receiver>listofselectedfield</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>312</x>
     <y>114</y>
    </hint>
    <hint type="destinationlabel" >
     <x>532</x>
     <y>272</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbscreen</sender>
   <signal>toggled(bool)</signal>
   <receiver>listofallfield</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>296</x>
     <y>87</y>
    </hint>
    <hint type="destinationlabel" >
     <x>291</x>
     <y>202</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbscreen</sender>
   <signal>toggled(bool)</signal>
   <receiver>listofselectedfield</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>461</x>
     <y>93</y>
    </hint>
    <hint type="destinationlabel" >
     <x>545</x>
     <y>204</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listofallfield</sender>
   <signal>currentItemChanged(QListWidgetItem*,QListWidgetItem*)</signal>
   <receiver>listofselectedfield</receiver>
   <slot>clearSelection()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>360</x>
     <y>252</y>
    </hint>
    <hint type="destinationlabel" >
     <x>478</x>
     <y>248</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listofselectedfield</sender>
   <signal>currentItemChanged(QListWidgetItem*,QListWidgetItem*)</signal>
   <receiver>listofallfield</receiver>
   <slot>clearSelection()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>513</x>
     <y>310</y>
    </hint>
    <hint type="destinationlabel" >
     <x>351</x>
     <y>323</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
